### DISCUSSION TITLE
Becuming by @Studio-Almond

### DISCUSSION THREAD

**Discussion by haroldao** [EvgeniyMukhamedjanov/liquid-ajax-cart#40] [Show and tell]
I just saw this website (by @Studio-Almond) using LAC on [Awwwards](https://www.awwwards.com/sites/becuming)

=> https://becuming.me/

**Comment by EvgeniyMukhamedjanov** [comment]
Wow, it is not possible to miss, how could I  :-)) 
Thank you for sharing!
If you have anything more with LAC - let me know as well, I'm going to update the "build with LAC" section soon.

**Comment by haroldao** [comment]
LAC spotted here too : https://alephbeauty.com (by Studio Almond too)

**Reply by EvgeniyMukhamedjanov** [reply]
@haroldao Really appreciate the update!
I've finally updated the "Built with LAC" section, added your Natz drinks website as well. :-)
